modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/spi/model/index/contributor/CPDefinitionModelDocumentContributor.java

```diff
@@ -261,10 +261,18 @@ public void contribute(Document document, CPDefinition cpDefinition) {
 			document.addKeyword(
 				CPField.SPECIFICATION_VALUES_NAMES,
 				TransformUtil.transformToArray(
-					_getFilteredCPDefinitionSpecificationOptionValues(
-						cpDefinitionSpecificationOptionValues),
-					CPDefinitionSpecificationOptionValue ->
-						CPDefinitionSpecificationOptionValue.getValue(
+					cpDefinitionSpecificationOptionValues,
+					cpDefinitionSpecificationOptionValue ->
+						com.liferay.portal.kernel.util.StringUtil.toLowerCase(
```

Comment:
not sure the toLowerCase method is needed

Reply:
I put toLowerCase as the filter always works in lowercase (reason why we put true on the other `addKeyword`). The reason why I put it manually and that passing multiple values we don't have the third params to specify lowercase true / false